lnbits/extensions/boltcards/models.py

```diff
@@ -45,6 +46,7 @@ async def lnurlpay_metadata(self) -> LnurlPayMetadata:
 
 class CreateCardData(BaseModel):
     card_name: str = Query(...)
+    message: str = Query("Boltcard (refund address {refund_url})")
```

Comment:
```suggestion
    message: str = Query("")
```

Reply:
I think it's more logical to have the message property always be representative of the actual message that gets sent. It's clearer and easier to inspect.

With the suggested changes, if someone wanted to actually make the message blank, they'd probably be surprised by the default message being used.

Reply:
Strings should not be initialized here like that. Would be better in lnurl.py